setup.py

```diff
@@ -79,7 +79,6 @@
         "data.logs_model.logs_producer",
         "data.model",
         "data.model.oci",
-        "data.queue",
```

Comment:
this might break the service tool, any reason we're removing this?

Reply:
if it's too much effort we can just merge this and deal with the service tool later

Reply:
oh odd you're right, i wonder why the sub packages are listed as well. nvm we should be fine then